src/rollback_to_stable/rts_btree.c

```diff
@@ -92,6 +92,18 @@ __rts_btree_abort_update(WT_SESSION_IMPL *session, WT_ITEM *key, WT_UPDATE *firs
      * store, it will have a different stop time point.
      */
     if (stable_upd != NULL) {
+        /*
+         * During recovery, there shouldn't be any updates in the update chain except when the
+         * updates are from a prepared transaction. Reset the transaction ID of the stable update
+         * that was restored as part of the unstable prepared tombstone.
+         */
```

Comment:
I am not a fan of this comment.
From the code, I don't think the reader can tell we are dealing with prepared data in the update chain and that there is an unstable prepared tombstone. It could also be due to my lack of knowledge in this area but I struggle to connect the comment and the code.

Reply:
The code makes more sense now.